k8sfv/run-test

```diff
@@ -97,20 +100,28 @@ while ! curl -k ${k8s_api_endpoint}/apis/extensions/v1beta1/thirdpartyresources;
     sleep 2
 done
 
-# Run Felix.
+# Get the API server's (self-signed) TLS certificate.
 mkdir -p k8sfv/output
+docker cp ${prefix}-apiserver:/var/run/kubernetes/apiserver.crt k8sfv/output/
+
+# Run Felix.
+#
+# Specifying CALICO_DATASTORE_TYPE as well as FELIX_DATASTORETYPE is
+# only needed for Felix 2.1 and earlier.  For 2.2, FELIX_DATASTORETYPE
+# is sufficient.
 docker run --detach --privileged --name ${prefix}-felix \
+       -e CALICO_DATASTORE_TYPE=kubernetes \
        -e FELIX_LOGSEVERITYSCREEN=${FELIX_LOG_LEVEL} \
        -e FELIX_DATASTORETYPE=kubernetes \
        -e FELIX_PROMETHEUSMETRICSENABLED=true \
        -e FELIX_USAGEREPORTINGENABLED=false \
        -e FELIX_DEBUGMEMORYPROFILEPATH="heap-<timestamp>" \
        -e K8S_API_ENDPOINT=${k8s_api_endpoint} \
-       -e K8S_INSECURE_SKIP_TLS_VERIFY=true \
+       -e K8S_CA_FILE=/testcode/k8sfv/output/apiserver.crt \
```

Comment:
Actually this is related, because Felix 2.1.x doesn't honour K8S_INSECURE_TLS_VERIFY. So to enable targeting 2.1.x we have to do certificate verification properly.